content/hi/partners/_index.html

```diff
@@ -0,0 +1,88 @@
+---
+title: सहभागी
+bigheader: कूबरनेट्स सहभागी
+abstract: कूबरनेट्स इकोसिस्टम का विस्तार.
+class: gridPage
+cid: सहभागी
+---
+
+<section id="users">
+    <main>
+        <h5>कुबरनेट्स सहयोगियों संग एक मज़बूत कोड बेस का निर्माण करता है जो संपूरक मंचों के स्पेक्ट्रम को सहयोग करता है।</h5>
+        <div class="col-container">
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>कुबरनेट्स प्रमाणित सेवा प्रदाता</b>
+              </h5>
+              <br>उद्यमों द्वारा कुबरनेट्स सफलतापूर्वक अपनाने में सहभागी अनुभवी सेवा प्रदाता।
+              <br><br><br>
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">KCSP सहभागियों को देखें</button>
+              <br><br><a href="https://www.cncf.io/certification/kcsp/">KCSP बनने में दिलचस्प</a>?
+            </center>
+          </div>
+          <div class="col-nav">
+            <center>
+              <h5>
+                <b>प्रमाणित कूबरनेट्स Distributions, Hosted Platforms, तथा Installers</b>
+              </h5>सॉफ्टवेयर के अनुपालन से ये सुनिश्चित किया जाता है कि प्रत्येक विक्रेता का संस्करण जरूरी API का समर्थन करे।
```

Comment:
1. `अनुपालन` doesn't add up in this sentence.
2. `संस्करण` means edition, but we need to use the word **version**

I think the ending of the sentence should be `कूबरनेट्स के जरूरी API का समर्थन करे।`

Reply:
Thanks for the PR!!!! Please address the suggestions that @mittalyashu  made and we can get this merged....

Reply:
1. Actually  `अनुपालन` here is used to make the meaning of the sentence more like "with fully satisfying the dependencies of software, this is ensured that every vendor's( `विक्रेता` ) version of kubernetes supports the required API's". Please tell me if there is any problem in this one.
2. `कूबरनेट्स के जरूरी API का समर्थन करे।` This one I missed, will change in next commit, thanks.

Reply:
@Rajakavitha1 What does vendor means exactly? Whom does it represent?

Maybe based on that we can figure out a different word?

Reply:
Yeah, that would be of great help!!